### DISCUSSION TITLE
i3 to i4 config migration tool

### DISCUSSION THREAD

**Discussion by tobeypeters** [i3/i3#5975] [Q&A]
I was making changes to make i3 configuration and I accidentally deleted the # in front of comment I had in it.  So, when I logged in, I got the message that I had errors in my config.  Like normal.   So, I clicked on the show errors thing and it said that I needed/should convert my config.  I'm on the latest and never saw anything before saying I needed to migrate it.  I ran the commands it gave me.  It seemed to work.  But, I get to messages:

Smartmatch is experimental at /usr/bin/i3-migrate-config-to-v4 line 113.
Smartmatch is experimental at /usr/bin/i3-migrate-config-to-v4 line 195.

```
#113: for_window [class="Netflix"] sticky enable
#195: $mod+F2 exec krunner

```
Don't think, it's talking about my config.  I assume, just ignore the messages?

**Comment by stefanorossiti** [comment]
You probably edited with some command for an older version of the i3 config, so i3 started the conversion script.

[man](https://man.archlinux.org/man/extra/i3-wm/i3-migrate-config-to-v4.1.en)

**Comment by orestisfl** [comment]
Putting
```
# i3 config file (v4)
```
on the top of your config will inhibit the conversion script
